modules/fluid_properties/unit/src/SodiumSaturationFluidPropertiesTest.C

```diff
@@ -0,0 +1,148 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "SinglePhaseFluidPropertiesTestUtils.h"
+#include "SodiumSaturationFluidPropertiesTest.h"
+
+/**
+ * Test that the fluid name is correctly returned
+ */
+TEST_F(SodiumSaturationFluidPropertiesTest, fluidName) { EXPECT_EQ(_fp->fluidName(), "sodium"); }
+
+/**
+ * Test that the molar mass is correctly returned
+ */
+TEST_F(SodiumSaturationFluidPropertiesTest, molarMass)
+{
+  ABS_TEST(_fp->molarMass(), 22.989769E-3, REL_TOL_SAVED_VALUE);
+}
+
+/**
+ * Test that the thermal conductivity and its derivatives are
+ * correctly computed.
+ */
+TEST_F(SodiumSaturationFluidPropertiesTest, thermalConductivity)
+{
+  const Real T = 800.0;
+  const Real p = 101325;
+
+  // Fink and Leibowitz (1979) list conductivity as 66.8 at 800 K, so the fit agrees well
+  REL_TEST(_fp->k_from_p_T(p, T), 66.9752096, REL_TOL_SAVED_VALUE);
```

Comment:
If you are comparing to a value that was computed (or measured) by something (someone) else, you should be using `REL_TOL_EXTERNAL_VALUE`. 

`REL_TOL_SAVED_VALUE` is used for: "I implemented this, I think this is the right value, so I am gonna compare to it".

There is some in-code doco in `FluidPropertiesTestUtils.h`.

Reply:
I hope the comment isn't misleading - that's just there for my own sanity check. The actual values that I'm testing against were computed with a Python script where I coded up the same correlations (with the idea thinking that it's probably unlikely I code up the exact same typo 2x).

What comparison value would you recommend for that? The `REL_TOL_SAVED_VALUE` failed a bunch of the density and specific volume tests by ~1e-8.

Reply:
I see, you are not comparing to Fink and Lebowitz value. But, again if the value was computed by something else it should be `REL_TOL_EXTERNAL_VALUE` which is 1e-3, so fairly loose.

`REL_TOL_SAVED_VALUE` is 1e-12. That's on purpose. You can compare to the value computed by this code and use `REL_TOL_SAVED_VALUE` - there is no problem with that, but then the comments are misleading. As we witnessed :-)